src/service/weather-api.jsx

```diff
@@ -0,0 +1,28 @@
+const BASE_URL = 'http://api.openweathermap.org/data/2.5/';
+const API_KEY = '52d1b8a08da4b0b2a6a50290af58193b';
+
+export const fetchCurrentWeather = async (city = 'Minsk') => {
+  const url = `${BASE_URL}weather?q=${city}&units=metric&appid=${API_KEY}`;
+
+  const response = await fetch(url);
+  const fetchObject = await response.json();
+
+  if (fetchObject.cod === 200) {
+    return fetchObject;
+  }
+
+  return Promise.reject(new Error(`Error not found`));
+};
+
+export const fetchForecastWeather = async (city = 'Minsk') => {
+  const url = `${BASE_URL}forecast?q=${city}&units=metric&appid=${API_KEY}`;
+
+  const response = await fetch(url);
+  const fetchObject = await response.json();
+
+  if (fetchObject.cod === '200') {
+    return fetchObject;
+  }
+
+  return Promise.reject(new Error(`Error not found`));
```

Comment:
same, don't forget about handling all statuses or figure it out how to do this in better way

Reply:
Ok

Reply:
Already answered this question